runtime/foucoco/src/lib.rs

```diff
@@ -973,6 +995,80 @@ impl orml_tokens_management_extension::Config for Runtime {
 	type AssetDeposit = AssetDeposit;
 }
 
+pub struct AllowedCurrencyIdVerifierImpl;
+impl treasury_buyout_extension::AllowedCurrencyIdVerifier<CurrencyId>
+	for AllowedCurrencyIdVerifierImpl
+{
+	fn is_allowed_currency_id(currency_id: &CurrencyId) -> bool {
+		matches!(
+			currency_id,
+			CurrencyId::XCM(0) | CurrencyId::XCM(1) | CurrencyId::XCM(2) | CurrencyId::XCM(6)
```

Comment:
Should we just allow all assets? Doesn't matter for the benchmarks but is less error prone

Reply:
Scratch that, I forgot that this setting is actually important for our runtime. When I wrote the comment I assumed that it's mostly used in the benchmark.